flask-server/main.py

```diff
@@ -377,16 +377,15 @@ def fetch_write_clinks():
 @app.route('/apis/fetch-bookmarks', methods=['GET'])
 def fetch_bookmarks():
     resp_token = decode_auth_token(request.headers.get('Authorization'))
-
+    clink_id = request.args.get('id')
+    
     if is_valid_instance(resp_token):
-        clink_id = request.headers.get('id')
+        bookmark_query = datastore_client.query(kind='bookmark').add_filter('creator', '=', str(resp_token)).add_filter('deleted', '=', False)
+        bookmark_query.order = ['created']
+        all_list = list(bookmark_query.fetch())
         if clink_id == 'All':
```

Comment:
Can the id be ALL or the title will be ALL?

Reply:
Both are set to All